interface/fax/fax_dispatch.php

```diff
@@ -156,7 +157,7 @@ function mergeTiffs()
             $info_msg .= mergeTiffs();
             // The -j option here requires that libtiff is configured with libjpeg.
             // It could be omitted, but the output PDFs would then be quite large.
-            $tmp0 = exec("tiff2pdf -j -p letter -o " . escapeshellarg($target) . " " . escapeshellarg($faxcache.'/temp.tif'), $tmp1, $tmp2);
+            $tmp0 = exec("tiff2pdf -p letter -F -o " . escapeshellarg($target) . " " . escapeshellarg($faxcache.'/temp.tif'), $tmp1, $tmp2);
```

Comment:
Sure, @stephenwaite, I did not find -j to make any substantive difference in pdf size.  I needed to add -F because at times the conversion wouldn't fill the whole pdf page.  I considered getting rid of tiff2pdf altogether here and going with gm convert (I've found tiff2pdf to be the weak link in the whole thing because of its trouble handling higher-bit tifs), but it seemed to work fine.